README.md

```diff
@@ -40,6 +40,14 @@ We literally have [*hundreds of terraform modules*][terraform_modules] that are
 
 
 
+## Usage
+
+
+**IMPORTANT:** The `master` branch is used in `source` just as an example. In your code, do not pin to `master` because there may be breaking changes between releases.
```

Comment:
@parabolic  thanks again.
Sorry, please remove the note that you added manually.
We updated `build-harness` to generate that note automatically.
After you remove it, please run

```
make init
make readme/deps
make readme
```

thanks

Reply:
Hi @aknysh that won't work, what I did is, I checked the build-harness template here at master,
https://raw.githubusercontent.com/cloudposse/build-harness/master/templates/README.md
and saw that if I don't have a `usage:` block in the README.yaml I won't get the note, hence my last change here adds that https://github.com/cloudposse/terraform-aws-cloudwatch-agent/pull/1/commits/3be501664d4e566e3255caa848a5f59122cb3779#diff-ef22b8f4f3d3aa4da77d8e2297dbb1fcR103
```
{{ if has (ds "config") "usage" }}
## Usage

{{ if (file.Exists "main.tf") }}
**IMPORTANT:** The `master` branch is used in `source` just as an example. In your code, do not pin to `master` because there may be breaking changes between releases.
Instead pin to the release tag (e.g. `?ref=tags/x.y.z`) of one of our [latest releases]({{ printf "https://github.com/%s/releases" (ds "config").github_repo}}).
{{end}}

{{ (ds "config").usage -}}
{{ end }}
```

If this is not intended then we will need to move the important note out of the `if` so that it's always present.

Reply:
As agreed separately , `examples` will be removed and instead `usage` will be used.